index.html

```diff
@@ -15,64 +15,159 @@
+          </div>
+          <div class="card">
+            <img src="l8istsh9ny.jpg" alt="">
+            <h4>Posted by Ryan Day</h4>
+            <h3>Event-Driven Software Delivery</h3>
+            <p>
+              Atomist co-founder and Chief Architect, Jim Clark, recently sat in with Rob Hirschfeld 
+              for an episode of the L8ist Sh9y Podcast. It's a wide-ranging conversation about delivery 
+              as workflow, event-driven delivery, code analysis, GraphQL and ChatOps.
+            </p>
+            <button>Read article</button>
+          </div>
+          <div class="card">
+            <img src="Podcast-Makers-1.jpg" alt="">
+            <h4>Posted by Ryan Day</h4>
+            <h3>Remove What You Hate 😡 from Software Development</h3>
+            <p>
+              My co-founder, Rod Johnson recently talked with Gene Kim, (see “The Phoenix Project” and 
+              “The DevOps Handbook”), and Alex Williams of The New Stack on The Makers podcast.
+            </p>
+            <button>Read article</button>
+          </div>
+          <div class="card">
+            <img src="tiny-graph-1.jpg" alt="">
+            <h4>Posted by Jessica Kerr</h4>
+            <h3>Problems you shouldn't have</h3>
+            <p>
+              The other day I heard a sad story about a build that was broken for weeks due to a bug 
+              introduced in an open-source library. It took the maintainer a long time to fix it 
+              (they're not getting paid for this, no criticism). In the enterprise they had so many 
+              layers of repository caching the team couldn't escape the buggy version. Even sadder, 
+              that library existed for fancy terminal output of a library that produced documentation 
+              (typedoc). A nonessential feature of a nonessential build step blocked 
+            </p>
+            <button>Read article</button>
+          </div>
+          <div class="card">
+              <img src="Image-from-iOS--44-.jpg" alt="">
+              <h4>Posted by John Doe</h4>
+              <h3>Lorem ipsum dolor sit amet consectetur.</h3>
+              <p>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Vitae aliquam odio molestias
+                doloribus consequuntur ipsa labore ab consectetur suscipit esse est quam, enim doloremque 
+                facere maxime velit error, ducimus temporibus.    
+              </p>
+              <button>Read article</button>
+            </div>
         </div>
       </div>
 
 
       <!-- Ready to try section -->
       <div class="row">
-        <div class="column">
-
+        <div class="col">
+          <h1>Ready to try Atomist?</h1>
+          <p>Go from brand new project to deployed in minutes.</p>
+          <button>Get Started for Free</button>
         </div>
       </div>
 
 
       <!-- Footer section -->
       <div class="row">
-        <div class="column">
-
+        <div class="col">
```

Comment:
I would have divided this section into five small sections, so it would contain 5 `<div class="row"> <div class="col">`. It's easier to manipulate and customize each section later. The sections I see are: 
 - logo
- nav links
- social media links
- copyright
- terms-privacy

Reply:
I'll take your advice here and change it

Reply:
Great! All the other changes are perfect, just one thing here, these 5 rows should be enclosed in a "parent" row-column pair, but we could do that in the pair programming session if you like.

Reply:
The pair programming session would be great